pom.xml

```diff
@@ -407,6 +408,18 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.owasp</groupId>
+                <artifactId>dependency-check-maven</artifactId>
```

Comment:
Can we add a skip parameter to the quick profile for this plugin?

Reply:
This plugin has only the configuration here, without an execution defined. So it should not run until executed manually. Therefore, I think we don't need to add a skip parameter.

Reply:
Right, I didn't realize it doesn't have a default execution. It is ok as is then.